com.endlessm.photos.json

```diff
@@ -1,12 +1,14 @@
 {
     "app-id": "com.endlessm.photos",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.36",
+    "runtime-version": "42",
     "sdk": "org.gnome.Sdk",
     "command": "eos-photos",
     "finish-args": [
-        "--share=ipc", "--share=network",
-        "--socket=x11", "--socket=wayland",
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
```

Comment:
I think we can change this to `fallback-x11`.

Reply:
Yes, please change to `fallback-x11`